changelog/@unreleased/pr-526.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
```

Comment:
@bmoylan autorelease-bot is failing to parse this file. Can you double-check the content and then also rename the file to be `.v2.yaml`?